app/microsoft/service.py

```diff
@@ -55,38 +55,42 @@ def remove_user_from_case(user, issue):
     Delete the permissions that a User has for a specific case (folder).
     """
     api = MSGraphAPI()
+
     case_path = f"cases/{issue.id}"
-    permissions = _get_permissions_for_paralegal(api, user, issue)
```

Comment:
So this `_get_permissions_for_paralegal` function (very difficult to read and understand) was written with the incorrect assumption that we need sharing URLs when our implementation is independent of them. Once we forget about sharing URLs, we can also remove the function since it isn't needed, and greatly simplify some of the code in our service functions, it also works on user testing.